docs/tutorials/first_steps/01_create_project.rst

```diff
@@ -29,42 +29,4 @@ Click on the **Create project** button (5).
     :align: center
     :alt: Create a new project
 
```

Comment:
I noticed that in the screenshot, the template is `Renku / Basic Python (3.7) Project`,  but it is Python 3.8 in RenkuLab now.

Reply:
fixed